src/main/java/com/revature/chronicle/controller/NoteController.java

```diff
@@ -112,4 +122,29 @@ public NoteController (NoteService ns, NoteRepo nr, TagRepo tr) {
         Optional<Note> targetNote = noteService.findById(id);
         return targetNote.map(note -> new ResponseEntity<>(note, HttpStatus.OK)).orElseGet(() -> new ResponseEntity<>(HttpStatus.BAD_REQUEST));
     }
+    
+    //Verify User Authorization
+    @PostMapping(path = "/whitelist/add/{noteId}", consumes = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
I don't think we need the 'add' in our uri since we already know its a post method.